autopush/tests/test_integration.py

```diff
@@ -508,6 +506,70 @@ def legacy_endpoint(self):
         self._settings._notification_legacy = False
 
 
+class SSLEndpointMixin:
```

Comment:
was actually avoiding it because I assumed twisted's TestCase was an old style class (not sure it would have even have mattered), now I see it's not, good catch